packages/mui-material/src/Select/SelectInput.js

```diff
@@ -10,17 +10,15 @@ import refType from '@mui/utils/refType';
 import ownerDocument from '../utils/ownerDocument';
 import capitalize from '../utils/capitalize';
 import Menu from '../Menu/Menu';
-import {
-  nativeSelectSelectStyles,
-  nativeSelectIconStyles,
-} from '../NativeSelect/NativeSelectInput';
+import { StyledSelectSelect, StyledSelectIcon } from '../NativeSelect/NativeSelectInput';
```

Comment:
Any idea why it was implemented like this before?

Reply:
I think it's because it is doable with Emotion, where you can reuse styles instead of components.

```js
const styles = {…};

const ComponentA = styled('div')(styles);

const ComponentB = styled('div')(styles);
```

But it would not work in Pigment (I guess), so need to use components instead which gives the same result:

```js
const ComponentA = styled('div')({ … });

const ComponentB = styled(ComponentA);
```